src/domain/mathExercise.ts

```diff
@@ -0,0 +1,103 @@
+import { evaluate } from "mathjs";
+import { Category, Exercise } from ".";
+import { v4 as uuidv4 } from 'uuid';
+
+interface params {
+    randomNumber?:Function,
+    noDivision?: boolean,  
+    highDigit?: number,
+    numberOfDigits?: number
+}
+
+
+export class MathExercise implements Exercise { 
+    symbols:string[] = []
+    noDivision: boolean | undefined;
+    highDigit: number;
+    numberOfDigits: number;
+    randomNumber: Function | undefined;
+    category: Category =  Category.WALK;
+    id: string;
+    correct?: boolean | undefined;
+    solution:Number;
+    exercise: string;
+    
+    constructor(opts?: params ) {
+        // Default values 
+        const defaults = {
+          randomNumber:() => Math.random(),
+          noDivision: true,  
+          highDigit: 6,
+          numberOfDigits: 4,
+        };     
+        this.id=uuidv4();
+        this.noDivision=opts && opts.noDivision? opts.noDivision: defaults.noDivision;
+        this.highDigit=opts&& opts.highDigit? opts.highDigit: defaults.highDigit;
+        this.numberOfDigits=opts&& opts.numberOfDigits?opts.numberOfDigits: defaults.numberOfDigits;
+        this.randomNumber=opts && opts.randomNumber? opts.randomNumber: defaults.randomNumber;
+        this._generate(this.randomNumber);   
+        this.solution = this._solution(); 
+        this.exercise = this.symbols.join(" ");
+      }
+
+    
+    _solution(): number {     
+        return evaluate(this.toString());
+    } 
+
+    _generate(randomNumber: any) {
+
+        const randomOperator = (): string =>
```

Comment:
This one should probably return and operator instead of a string? Also the logic should use operator enums.